docs/src/catalyst_applications/nonlinear_solve.md

```diff
@@ -0,0 +1,99 @@
+```
+Finally, we can solve it using the `solve` command, returning the steady state solution:
+```@example nonlinear_solve1
+using NonlinearSolve
+solve(nl_prob)
+```
+
+NonlinearSolve provides [a wide range of potential solvers](https://docs.sciml.ai/NonlinearSolve/stable/solvers/NonlinearSystemSolvers/). If we wish to designate one, it can be supplied as a second argument to `solve`. Here, we use the Newton Trust Region method:
+```@example nonlinear_solve1
+solve(nl_prob, TrustRegion())
+nothing # hide
+```
+
+## [Finding steady states through ODE simulations](@id nonlinear_solve_ode_simulation_based)
+The `NonlinearProblem`s generated by Catalyst corresponds to ODEs. A common method of solving these is to simulate the ODE from an initial condition until a steady state is reached. NonlinearSolve supports this through the `DynamicSS` method. To use it, an appropriate ODE solver (and any options you wish it to use) must also be supplied (with [a large number being available](https://docs.sciml.ai/DiffEqDocs/stable/solvers/ode_solve/)). Here, we will use the `Tsit5` ODE solver to find the steady states of our dimerisation system.
+```@example nonlinear_solve1
+using OrdinaryDiffEq, SteadyStateDiffEq
+solve(nprob, DynamicSS(Tsit5()))
+```
+Here, we had to import [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl) (to use `Tsit5`) and [SteadyStateDiffEq.jl](https://github.com/SciML/SteadyStateDiffEq.jl) (to use `DynamicSS`).
+
+Like previously, the found steady state is unaffected by the initial `u` guess. However, when [finding the steady states of systems with conservation laws](@ref nonlinear_solve_conservation_laws) it is important to select a `u` guess corresponding to the initial condition of the ODE simulation which steady state is sought.
+
+
+## [Systems with conservation laws](@id nonlinear_solve_conservation_laws)
+As described in the section on homotopy continuation, when finding the steady states of systems with conservation laws, [additional considerations have to be taken](homotopy_continuation_conservation_laws). E.g. consider the following two-state system:
+```@example nonlinear_solve2
+using Catalyst, NonlinearSolve # hide
+two_state_model = @reaction_network begin
+    (k1,k2), X1 <--> X2
+end
+```
+It has an infinite number of steady states. To make steady state finding possible, information of the system's conserved quantities (here $C=X1+X2$) must be provided. Since these can be computed from system initial conditions (`u0`, i.e. those provided when performing ODE simulations), designating an `u0` is often the best way. This can either be done by performing [ODE simulation-based steady state finding](@ref), using the initial condition as the initial `u` guess. Alternatively, any conservation quantities can be eliminated when the `NonlinearProblem` is created. This feature is supported by Catalyst's [conservation law finding and elimination feature](@ref network_analysis_deficiency).
```

Comment:
```
This can either be done by performing [ODE simulation-based steady state finding](@ref), using the initial condition as the initial `u` guess. Alternatively, any conservation quantities can be eliminated when the `NonlinearProblem` is created.
```
is a bit awkwardly/confusingly worded. I would reword.

Reply:
👍